src/api/OrderAPI.ts

```diff
@@ -278,11 +269,41 @@ export class OrderAPI {
 
   /* ============ Private Assertions ============ */
 
+  private assertFillAmountValidity(
+    signedIssuanceOrder: SignedIssuanceOrder,
+    quantityToFill: BigNumber,
+    orders: ExchangeOrder[],
+  ): void {
+    let zeroExFillAmounts = SetProtocolUtils.CONSTANTS.ZERO;
```

Comment:
Are we tallying the total maker token used? If so, we can rename to something more descriptive like `makerTokenRequiredQuantity`.